assets/js/modules/ads/pax/services.test.js

```diff
@@ -267,5 +267,16 @@ describe( 'PAX partner services', () => {
 				} );
 			} );
 		} );
+
+		describe( 'termsAndConditionsService', () => {
+			it( 'notify callback should return an empty object', async () => {
+				const termsAndConditionsServiceNotifyResponse =
+					await services.termsAndConditionsService.notify();
+
+				expect( termsAndConditionsServiceNotifyResponse ).toMatchObject(
```

Comment:
This can be more strict for now
```suggestion
				expect( termsAndConditionsServiceNotifyResponse ).toEqual(
```